roles/kubernetes-apps/ingress_controller/ingress_nginx/templates/role-ingress-nginx.yml.j2

```diff
@@ -28,23 +28,17 @@ rules:
     verbs: ["get", "list", "watch"]
   - apiGroups: ["coordination.k8s.io"]
     resources: ["leases"]
-    # Defaults to "<election-id>-<ingress-class>"
-    # Here: "<ingress-controller-leader>-<nginx>"
-    # This has to be adapted if you change either parameter
-    # when launching the nginx-ingress-controller.
+    # Defaults to "<election-id>", defined in 
+    # ds-ingress-nginx-controller.yml.js
+    # by a command-line argument.
+    # 
+    # This is the correct behaviour for ingress-controller
+    # version 1.8.0
```

Comment:
The comments may not need to change :-)
BTW: ds-ingress-nginx-controller.yml.js => ds-ingress-nginx-controller.yml.j2